core/modules/comment/comment.admin.inc

```diff
@@ -244,7 +77,7 @@ function comment_multiple_delete_confirm_submit($form, &$form_state) {
     watchdog('content', 'Deleted @count comments.', array('@count' => $count));
     backdrop_set_message(format_plural($count, 'Deleted 1 comment.', 'Deleted @count comments.'));
   }
-  $form_state['redirect'] = 'admin/content/comment';
+  $form_state['redirect'] = 'admin/content/comments';
```

Comment:
I don't think we should change the URL. The view path should remain `admin/content/comment`